Dockerfile

```diff
@@ -0,0 +1,35 @@
+# To build image:
+#
+#     % docker build --tag fam:latest .
+#
+# To run an existing playbook simply specify the path. Override any variables necessary using the
+# --extra-vars option.
+#
+#     % docker run -it --rm \
+#          fam:latest \
+#              --extra-vars "foreman_server_url=https://192.168.122.3" \
+#              /foreman-ansible-modules/test/test_playbooks/organization.yml
```

Comment:
You might also mention that users may need the `--network host` option. At least, I did. ;)